packages/story-editor/src/utils/pointerEventsCss.ts

```diff
@@ -19,8 +19,12 @@
  */
 import { css } from 'styled-components';
 
+interface pointerEventsCssProps {
+  pointerEvents?: string;
```

Comment:
```suggestion
  pointerEvents?: 'none' | 'auto' | 'initial';
```

Also makes the `typeof pointerEvents === 'string'` check below superfluous.